pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -563,9 +568,13 @@ func (m *cgroupManagerImpl) ReduceCPULimits(cgroupName CgroupName) error {
 
 func getStatsSupportedSubsystems(cgroupPaths map[string]string) (*libcontainercgroups.Stats, error) {
 	stats := libcontainercgroups.NewStats()
-	for _, sys := range getSupportedSubsystems() {
+	for sys, required := range getSupportedSubsystems() {
 		if _, ok := cgroupPaths[sys.Name()]; !ok {
-			return nil, fmt.Errorf("Failed to find subsystem mount for subsystem: %v", sys.Name())
+			if required {
+				return nil, fmt.Errorf("Failed to find subsystem mount for required subsystem: %v", sys.Name())
+			}
+			// the cgroup is not mounted, but its not required so continue...
+			continue
```

Comment:
Can we please log that we are continuing even though the cgroup is missing?

Reply:
this gets called a lot, so i want to avoid excessive logging calls.  at best it would have had to been logged at a really high level.

Reply:
+1 to avoid excessiveness.  yes, please to log once at high level (and say what aspect will not work because a cgroup is missing)